### DISCUSSION TITLE
Can't get xaml intellisense in VSCode

### DISCUSSION THREAD

**Discussion by TungLamC** [unoplatform/uno#13858] [Q&A] [answered]
I currently start use Uno Platform, everything is nice, but VSCode xaml intellisense never work. I have already tried to disable C# dev kit extensions, enable `dotnet.server.useOmnisharp` settings and select project in omnisharp, C# intellisense can work, include thoes properties define in xaml by x:Name, but xaml intellisense can't.

**my uno-check**
> I don't need mobile platform, so I don't fix errors about Android.

![image](https://github.com/unoplatform/uno/assets/29020158/8e6c017f-63b9-416c-b623-8cc10e9a1749)

**Uno Platform Output**
![image](https://github.com/unoplatform/uno/assets/29020158/f07e6835-d7b6-4c58-a3e0-ee3f20df5e96)

**VSCode statusbar**
![image](https://github.com/unoplatform/uno/assets/29020158/49264ee5-8f32-428c-b0a3-fa60b472388d)

The problem is `[Error] setLSActiveProject failed: Error: Starting server failed`, I dont have any idea to fix that, anybody can help me?

**Comment by sasakrsmanovic** [comment]
Maybe @spouliot would know?

**Reply by spouliot** [reply]
This looks like a bug 😢 where `setLSActiveProject` being called too early (before we know the targetFramework).

**Reply by spouliot** [reply]
Tracked in https://github.com/unoplatform/uno/issues/13881

**Reply by spouliot** [reply, marked as answer]
This was fixed and released last month.
